src/index.js

```diff
@@ -60,40 +60,39 @@ const displaySchedules = () => {
   const myImage = new Image();
   // eslint-disable-next-line camelcase
```

Comment:
Let's remove all disabled Linters, Linting makes your code look more cleaner and professional 🙂